bpf/kmesh/workload/sockops.c

```diff
@@ -193,7 +194,10 @@ int record_tuple(struct bpf_sock_ops *skops)
             break;
         if (bpf_sock_ops_cb_flags_set(skops, BPF_SOCK_OPS_STATE_CB_FLAG) != 0)
             BPF_LOG(ERR, SOCKOPS, "set sockops cb failed!\n");
-        enable_encoding_metadata(skops);
+        __u32 *current_sk = (__u32 *)skops->sk;
```

Comment:
the type of the the key in `map_of_dst_info` need to be changed to `u64` to avoid unnecessary conflict.